src/components/ProjectContents.jsx

```diff
@@ -18,9 +18,9 @@ class ProjectContents extends Component {
     return (
       <div>
         <h2>Project Contents</h2>
-        <p>Title: { project_contents.title }</p>
-        <p>Description: { project_contents.description }</p>
-        <p>Introduction: { project_contents.introduction }</p>
+        <p data-translation-project-contents={project_contents.title}>Title: { project_contents.title }</p>
```

Comment:
Should these data attributes be storing the names of the keys, rather than their values? Otherwise, I'm not seeing how the values get mapped back to keys when they're edited.